[23042022 16:48:34] dionaea /root/dionaea/src/dionaea.c:756-warning: Setgroups dropping groups failed
[23042022 16:48:34] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:78-warning: Starting cleanup loop
[23042022 16:49:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:50:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:51:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:54:35] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:55:50] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:56:06] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[23042022 16:56:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:56:06] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[23042022 16:56:06] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[23042022 16:56:06] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[23042022 16:56:06] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[23042022 16:56:06] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 16:56:06] connection /root/dionaea/src/connection.c:137-warning: getpeername failed (Transport endpoint is not connected)
[23042022 16:56:06] connection /root/dionaea/src/connection_tcp.c:97-warning: accepting connection failed, closing connection
[23042022 17:01:40] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 17:02:53] sip /opt/dionaea/lib/dionaea/python/dionaea/sip/__init__.py:45-warning: Cleanup
[23042022 17:02:53] python /root/dionaea/modules/python/module.c:1039-warning: AttributeError at 'SipSession' object has no attribute 'apply_parent_config'
